Frends.AmazonS3.UploadObject/Frends.AmazonS3.UploadObject/Definitions/Connection.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AmazonS3.UploadObject.Definitions
```

Comment:
You should use one-line namespace definition, so `namespace Frends.AmazonS3.UploadObject.Definitions;` and remove brackets that wrap the namespace. This was something that came up in one PR not long ago.

Reply:
This is still not a one-line namespace definition.